frameioclient/download.py

```diff
@@ -1,18 +1,49 @@
-import requests
-import math
 import os
+import math
+import requests
+
+from .exceptions import DownloadException
 
 class FrameioDownloader(object):
   def __init__(self, asset, download_folder):
     self.asset = asset
     self.download_folder = download_folder
 
+  def get_download_key(self):
+    try:
+      url = self.asset['original']
+    except KeyError:
+      if self.asset['is_session_watermarked'] == True:
+        resolution_map = dict() 
+        for resolution_key, download_url in sorted(self.asset['downloads'].items()):
+          resolution = resolution_key.split("_")[1] # Grab the item at index 1 (resolution)
+          try:
+            resolution = int(resolution)
+            if download_url is not None:
+              resolution_map.update({
+                resolution: download_url
+              })
+          except ValueError:
+            continue
+
+        # Grab the highest resolution now
+        url = resolution_map.values()[0]
```

Comment:
are we sure this grabs the highest resolution? maybe add some tests?
Also doesn't look like `resolution_map` needs to be a `dict`; if we're only using `values`?